src/miral/basic_window_manager.cpp

```diff
@@ -182,14 +197,16 @@ void miral::BasicWindowManager::remove_surface(
     std::weak_ptr<scene::Surface> const& surface)
 {
     Locker lock{this};
-    try
-    {
-        remove_window(session, info_for(surface));
-    }
-    catch (std::out_of_range const& ex)
+    if (app_info.find(session) == app_info.end())
     {
-        fatal_error("Could not find surface to remove in miral::BasicWindowManager::remove_surface()");
+        log_warning("Removed surface with unknwon session");
```

Comment:
"unknwon"?

Reply:
I will not mistype unknown
I will not mistype unknown
I will not mistype unknown